zeppelin-web/package.json

```diff
@@ -15,7 +15,7 @@
     "dev:helium": "HELIUM_BUNDLE_DEV=true webpack-dev-server --hot",
```

Comment:
HI @andersonjonathan, you have to use `dev:helium` instead of `visdev` in 0.8.0. It was `visdev`, but changed.